### PR TITLE
feat(run-protocol): collect runStake fees

### LINKED ISSUE #5004: RUNstake fees are wired up correctly

### PR DESCRIPTION
**Pull request by turadg**
## Description

- add `makeCollectFeesInvitation` to RunStake's creator
- fix types
- add runStakeCreator to the list in `startRewardDistributor`

### Security Considerations

RunStake creator facet exposes a way to collect its fees

fixes: #5004

### Documentation Considerations

--

### Testing Considerations

Relying on types. Open to requests for additional coverage.

### CODE REVIEW

File: packages/run-protocol/src/runStake/runStake.js [hunk 1]
```diff
@@ -197,13 +198,19 @@ export const start = async (
...
-    }),
-  );
+  /** @type {ERef<RunStakeCreator>} */
+  const creatorFacet = Far('runStake creator', {
+    provideAttestationMaker: att.creatorFacet.provideAttestationTool,
+    makeCollectFeesInvitation: () => {
+      return makeMakeCollectFeesInvitation(
+        zcf,
+        rewardPoolSeat,
+        debtMint.getIssuerRecord().brand,
```
Comment (Chris-Hibbert) [inline packages/run-protocol/src/runStake/runStake.js]:
```suggestion
        debtBrand,
```

**Review by Chris-Hibbert — APPROVED** [review]
LGTM

File: packages/run-protocol/src/runStake/runStake.js [hunk 2]
```diff
@@ -197,13 +198,19 @@ export const start = async (
...
+        'RUN',
+      ).makeCollectFeesInvitation();
+    },
+  });
 
-  // @ts-expect-error makeGovernorFacet loses type info
-  return { publicFacet, creatorFacet };
+  return { publicFacet, creatorFacet: makeGovernorFacet(creatorFacet) };
```
Comment (dckc) [inline packages/run-protocol/src/runStake/runStake.js:214]: Does this mean runStake governance was previously busted? Did we not have any tests for it?

**Review by dckc — COMMENTED** [review]
I defer to @Chris-Hibbert ; I'm fuzzy on `makeGovernorFacet(creatorFacet)`

File: packages/run-protocol/src/runStake/runStake.js (hunk 2 above)
Comment (turadg) [reply packages/run-protocol/src/runStake/runStake.js:214]: I just moved the `makeGovernorFacet` down here to make more obvious that there are two slightly different things named `creatorFacet`.

**Comment by dckc** [comment]
Darn; I think the permit for that needs tweaking:

```diff
modified   packages/run-protocol/src/core-proposal.js
@@ -137,6 +137,7 @@ const REWARD_MANIFEST = harden({
       loadVat: true,
       vaultFactoryCreator: true,
       ammCreatorFacet: true,
+      // TODO: runStake
       zoe: true,
     },
     produce: {
```

`test-boot.js` used to verify all the permits, but I think it only tests the platform bootstrap ones, not the run-protocol ones, now.

**Pull request merged**